acai-ios/ViewControllers/LoginViewController.swift

```diff
@@ -82,17 +76,32 @@ class LoginViewController: UITableViewController {
     }
 
     private func formatNavigationBar() {
+        let navigationTitleLabel = UILabel()
+        navigationTitleLabel.text = "Account"
+        navigationTitleLabel.font = UIFont.avenirNextMedium.withSize(24)
+        navigationTitleLabel.sizeToFit()
+        let navigationBarTitleItem = UIBarButtonItem(customView: navigationTitleLabel)
+        let navigationBarBackButton = UIBarButtonItem(image: UIImage(named: "backArrow"), style: .done, target: self, action: #selector(cancelled))
```

Comment:
i would change the target function to something like `backButtonPressed` so that its clearer